src/platform/packages/shared/response-ops/alerts-delete/components/descriptive_form_group.test.tsx

```diff
@@ -11,33 +11,51 @@ import React from 'react';
 import { render, screen, fireEvent } from '@testing-library/react';
 import { AlertDeleteDescriptiveFormGroup } from './descriptive_form_group';
 import * as i18n from '../translations';
-import { httpServiceMock } from '@kbn/core/public/mocks';
+import { httpServiceMock, notificationServiceMock } from '@kbn/core/public/mocks';
 import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
 import { __IntlProvider as IntlProvider } from '@kbn/i18n-react';
 
 const http = httpServiceMock.createStartContract();
+const notifications = notificationServiceMock.createSetupContract();
 
 describe('AlertDeleteRuleSettingsSection', () => {
-  const queryClient = new QueryClient();
+  const queryClient = new QueryClient({
```

Comment:
Let's add the logger to avoid polluting the tests with logs.

```
logger: {
    log: console.log,
    warn: console.warn,
    error: () => {},
  },
```

Reply:
We have noticed in the past that React Query was emitting errors on network errors that were polluting the CI. It is related to this https://tanstack.com/query/v4/docs/framework/react/guides/testing#turn-off-network-error-logging. Not sure if it is applicable anymore.